test/e2e/windows/hybrid_network.go

```diff
@@ -50,19 +50,27 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 	ginkgo.Context("for all supported CNIs", func() {
 
 		ginkgo.It("should have stable networking for Linux and Windows pods", func() {
-			ginkgo.By("creating linux and windows pods")
+
 			linuxPod := createTestPod(f, linuxBusyBoxImage, linuxOS)
+			ginkgo.By("creating a linux pod and waiting for it to be running")
 			linuxPod = f.PodClient().CreateSync(linuxPod)
+
 			windowsPod := createTestPod(f, windowsBusyBoximage, windowsOS)
+
 			windowsPod.Spec.Containers[0].Args = []string{"test-webserver"}
+			ginkgo.By("creating a windows pod and waiting for it to be running")
 			windowsPod = f.PodClient().CreateSync(windowsPod)
 
+			ginkgo.By("verifying pod external connectivity to the internet")
+
 			ginkgo.By("checking connectivity to 8.8.8.8 53 (google.com) from Linux")
 			assertConsistentConnectivity(f, linuxPod.ObjectMeta.Name, linuxOS, linuxCheck("8.8.8.8", 53))
 
 			ginkgo.By("checking connectivity to www.google.com from Windows")
 			assertConsistentConnectivity(f, windowsPod.ObjectMeta.Name, windowsOS, windowsCheck("www.google.com"))
 
+			ginkgo.By("verifying pod internal connectivity to the cluster dataplane")
+
 			ginkgo.By("checking connectivity from Linux to Windows")
```

Comment:
Why have two By statements together here?

Reply:
ah, yes, this is because the two by statements below it are associated with external internet connectivity....